sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/UploadBufferPool.java

```diff
@@ -28,8 +28,10 @@
  * to be available, and either store the incomplete buffer to be filled on the next write or return the filled buffer to
  * be sent. Filled buffers can be uploaded in parallel and should return buffers to the pool after the upload completes.
  * Once the source terminates, it should call flush.
+ *
+ * RESERVED FOR INTERNAL USE ONLY
```

Comment:
Will revert changes to this as well since now the encrypted client calls into blob client methods